src/components/Accordion/Accordion.test.tsx

```diff
@@ -87,13 +87,29 @@ describe('Accordion', () => {
 
 describe('subtitle', () => {
   it('should show subtitle when "subtitle" prop is set', () => {
-    render({ subtitle: 'Subtitle is here' });
+    render({
+      children: (
+        <>
+          <AccordionHeader subtitle='Subtitle is here'>
+            AccordionHeader
+          </AccordionHeader>
+          <AccordionContent>AccordionContent</AccordionContent>
+        </>
+      ),
+    });
 
     expect(screen.getByText('Subtitle is here')).toBeInTheDocument();
   });
 
   it('should not show subtitle when "subtitle" prop is not set', () => {
-    render({ subtitle: undefined });
+    render({
+      children: (
+        <>
+          <AccordionHeader>AccordionHeader</AccordionHeader>
+          <AccordionContent>AccordionContent</AccordionContent>
+        </>
+      ),
+    });
```

Comment:
Could consider to extract this to a separate test for the `AccordionHeader` component instead, as its not related to the rest of Accordion

Reply:
Yeah, but its so little, I thought it might be a bit overkill? 😅

Reply:
As I mentioned, I think it would make sense as separate test, because this is part of the interface of the `AccordionHeader`, and not something `Accordion` has any knowledge about. 

Splitting things into parts that belong together improves readabilty and discoverability - if I wanted to look into tests (or add new tests) for the `AccordionHeader`, I would expect to look in a file called `AccordionHeader.tests` and not `Accordion.test`. 

But as mentioned, its not important 🙂

Reply:
Important or not, it's done now anyway 😅